pkgs/by-name/se/seagoat/package.nix

```
@@ -0,0 +1,65 @@
+{
+  lib,
+  fetchPypi,
+  python3,
+  python3Packages,
+  ripgrep,
+}:
+
+python3Packages.buildPythonPackage rec {
```

Since we mainly care about the executable rather than the modules, it makes sense to use:

```suggestion
python3Packages.buildPythonApplication rec {
```

See [nixpkgs/doc/languages-frameworks/python.section.md](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonapplication-function-buildpythonapplication-function)